nodes/list/set_list_element.py

```diff
@@ -43,22 +43,29 @@ def drawAdvanced(self, layout):
 
     def getExecutionCode(self):
         yield "self.errorMessage = ''"
+        if self.isTuple: yield "_list = list(_list)"
```

Comment:
do you miss () here?
self.isTuple will always be true.
Another option would be to use the "@property" decorator on the isTuple function